apps/explorer/src/pages/transaction-result/transaction-summary/ObjectChanges.tsx

```diff
@@ -134,15 +134,44 @@ function ObjectDetailPanel({
     );
 }
 
+function getObjectDetailData(objectType: string) {
```

Comment:
I am not sure if we need to check `isCoin` here since this is the general objectType

```
pkgId::moduleName::type
```

Which means, even if it is a Coin, it would follow this, with the difference being `Coin<0x2::sui::SUI>` for the type part. We really only need to take out the 3 parts to get what we need, the Coin just has more :: in there. Let me know if you think otherwise. I can do a split for :: instead and just grab everything from splitArray.slice(2).join('::') also for the type part